frontend/src/js/controllers/challengeCreate.js

```diff
@@ -72,6 +74,7 @@
                     };
                 }
                 utilities.sendRequest(parameters, 'header', 'upload');
+                utilities.showLoader();
```

Comment:
Just move this above to ` utilities.sendRequest(parameters, 'header', 'upload');` @adamstafa

Reply:
@aka-jain Done.